var/spack/repos/builtin/packages/py-pytables/package.py

```diff
@@ -18,6 +18,7 @@ class PyPytables(PythonPackage):
     version('3.2.2', '7cbb0972e4d6580f629996a5bed92441', url="https://github.com/PyTables/PyTables/archive/v.3.2.2.tar.gz")
 
     depends_on('hdf5@1.8.0:')
+    depends_on('py-mock@2.0:', type=('build', 'run'))
```

Comment:
This dependency has been added here:
https://github.com/PyTables/PyTables/commit/b721f8b19f5e6e48a84bc80ba5c3fd8be95c6b93

Personally, I don't share the approach of adding test dependencies to runtime.
We can patch it out here in the recipe if you deem it preferable.

Reply:
Hm. Docs are a little sparse on python testing. `setup.py` should list mock as a test dependency, and pull it in during `python setup.py test` normally (spack supposedly executes this when doing `spack install --test=root <blah>`).

Reply:
Since this is nothing extreme, the change is OK with me.

Reply:
This missing dependency broke modules depending on `py-tables`; and since it was a runtime issue, CI was not able to catch it.
I wonder if it's worth adding some smoke test for the modules (like running `command --help` or smth like this) as a part of module deployment.

Reply:
We are thinking of linking some of our Jenkins CI with deployment to be sure end users are not affected but didn't get time. Also we have to see build times for PR.

Reply:
OK, thanks, good to know.